src/library/library.cpp

```diff
@@ -16,17 +16,7 @@
 #include "library/sidebarmodel.h"
 #include "library/trackcollection.h"
 #include "library/trackmodel.h"
-#include "library/crate/cratefeature.h"
-#include "library/rhythmbox/rhythmboxfeature.h"
-#include "library/banshee/bansheefeature.h"
-#include "library/recording/recordingfeature.h"
-#include "library/itunes/itunesfeature.h"
-#include "library/mixxxlibraryfeature.h"
-#include "library/autodj/autodjfeature.h"
-#include "library/playlistfeature.h"
-#include "library/traktor/traktorfeature.h"
 #include "library/librarycontrol.h"
-#include "library/setlogfeature.h"
```

Comment:
Why did you move these #includes?

Reply:
Moving these includes to the header resolved build errors like "not able to resolve `AutoDJFeature* m_pAutoDJFeature;`" in library.h for example. And I thought it's more consistent to have all feature includes in one place.

Reply:
Okay